Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM php:7.0-apache
+
+LABEL maintainer="Kitware, Inc. <software.kitware.com>"
```

Comment:
@opadron missing @ symbol here.

Reply:
Can we make this `cdash@public.kitware.com`?  That's our public mailing list for CDash, so it's a bit more specific.